client/my-sites/people/invite-people/index.jsx

```diff
@@ -436,10 +451,170 @@ class InvitePeople extends React.Component {
 		return inviteForm;
 	};
 
+	generateInviteLinks = () => {
+		return this.props.generateInviteLinks( this.props.siteId );
+	};
+
+	disableInviteLinks = () => {
+		if (
+			window.confirm(
+				this.props.translate( 'Are you sure you wish to disable these invite links?' )
+			)
+		) {
+			return this.props.disableInviteLinks( this.props.siteId );
+		}
+	};
+
+	showInviteLinkForRole = ( event ) => {
+		const { inviteLinks } = this.props;
+		const role = event.target.value || 'administrator';
+		this.setState( { activeInviteLink: inviteLinks[ role ] } );
+	};
+
+	getActiveInviteLink = ( activeInviteLink ) => {
+		if ( activeInviteLink ) {
+			return activeInviteLink;
+		}
+
+		if ( this.props.inviteLinks && typeof this.props.inviteLinks.administrator !== 'undefined' ) {
+			return this.props.inviteLinks.administrator;
+		}
+
+		return false;
+	};
+
+	onInviteLinkCopy = () => {
+		this.setState( {
+			showCopyConfirmation: true,
+		} );
+
+		clearTimeout( this.dismissCopyConfirmation );
+		this.dismissCopyConfirmation = setTimeout( () => {
+			this.setState( {
+				showCopyConfirmation: false,
+			} );
+		}, 4000 );
+	};
+
+	renderCopyLinkButton = ( link, className ) => {
+		const { translate } = this.props;
+
+		let label;
+		if ( this.state.showCopyConfirmation ) {
+			label = translate( 'Copied!' );
+		} else {
+			label = translate( 'Copy link', { context: 'verb' } );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **32** times:
`translate( 'Copy link' )` *ES Score: 12*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/invite-links)


:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.